kytos/core/switch.py

```diff
@@ -191,6 +192,24 @@ def as_json(self):
         return json.dumps(self.as_dict())
 
 
+class Connection_states(Enum):
+    """Enum of possible general connections states."""
+
+    NEW = 0
+    SETUP = 1
+    ESTABLISHED = 2
+    FINISHED = 3
+
+
+class Protocol_info():
```

Comment:
I think just ``Protocol`` is ok, specially if we add links to pack and unpack functions later.